compat/sched.h

```diff
@@ -79,5 +85,9 @@ CpuAnd(cpu_set_t *destset, const cpu_set_t *srcset1, const cpu_set_t *srcset2)
 }
 #endif /* CPU_AND */
 
-#endif /* SQUID_COMPAT_CPU_H */
+#endif
```

Comment:
Please provide a trailing comment to indicate which `#if` is closed here. I am not sure, but it may be HAVE_SYS_CPUSET_H.